tools/webots.py

```diff
@@ -100,7 +105,11 @@ def exec_build(target, roles, clean):
 
     # Compiles code for correct target
     print("Building code...")
-    exit_code = subprocess.run(["./b", "build"]).returncode
+    build_command = ["./b", "build"]
+    # Check if a -j value has been given
+    if jobs != None:
```

Comment:
Not a big deal, but just to be consistent with the rest of our codebase.
```suggestion
    if jobs:
```